st2common/st2common/cmd/apply_rbac_definitions.py

```diff
@@ -29,8 +29,6 @@
     'main'
 ]
 
-cfg.CONF.register_cli_opt(cfg.BoolOpt('verbose', short='v', default=False))
```

Comment:
This is anyways done inside script_setup.py. See description - I verified by removing this line we still see --verbose.